test-cases/gutenberg/unsupported-block-editing.md

```diff
@@ -35,68 +46,75 @@ A WP.com Simple site (i.e. not an Atomic site). Any free WP.com site should suff
+6. Edit the block content
 7. Tap the Continue button and expect to be taken back to the block editor
 8. Publish the post and verify it contains the edits
 
 
 ##### TC004
 
+**Known Issues**
+- [Editing unsupported blocks is not enabled on sites accessed via older WP.com accounts](https://github.com/wordpress-mobile/gutenberg-mobile/issues/3425)
+
 #### Precondition
 
 A WP.com Atomic site (i.e. a WP.com site with a Business plan) with the **Classic editor** set as the default. Here are the steps for creating this if necessary:
 
 1. Find a site with a Business plan or higher
-2. Install and activate the Classic Editor plugin
-3. In Calypso choose your site and go to Tools | Plugins | Installed Plugins, then tap Settings under the Classic Editor plugin and set **Default editor for all users** to **Classic Editor**
+2. Find an account created in 2019 or later (account age is found under https://wordpress.com/me/account), otherwise the known issue listed above will break the Unsupported Block Editor
+3. Install and activate the Classic Editor plugin
+4. In Calypso choose your site and go to Tools | Plugins | Installed Plugins, then tap Settings under the Classic Editor plugin and set **Default editor for all users** to **Classic Editor**
 
 ### Editing unsupported blocks is disallowed on Classic-enabled Atomic sites
 
-1. On the site described above, add a post, then add a block that's not yet supported on mobile (e.g. Audio at time of writing)
+1. On the site described above, add a post, then add a block that's not yet supported on mobile
 2. Log into the WordPress mobile app using the site's WP.com account
 3. Open the post and expect to see the block rendered as a placeholder with the text "Unsupported"
-4. Tap the `(?)` icon and expect to NOT see an option to edit the block in a web browser
+4. Tap the `(?)` icon and expect to **not** see an option to edit the block in a web browser
 
 
 ##### TC005
 
 **Known Issues**
-- A Jetpack-site recently connected to a WPCom account older than December 2018 won’t display the option to open the Unsupported Block Editor. ([wordpress-mobile/gutenberg-mobile#3425](https://github.com/wordpress-mobile/gutenberg-mobile/issues/3425))
+- [Editing unsupported blocks is not enabled on sites accessed via older WP.com accounts](https://github.com/wordpress-mobile/gutenberg-mobile/issues/3425)
 
 #### Precondition
 
 For this test, you need a self-hosted site with the Jetpack plugin installed, activated and connected to a WP.com account. Here are the steps for creating this if needed:
 
+1. Log into a WP.com account
 1. Create a site on https://jurassic.ninja/
-2. Connect the Jetpack plugin (it comes pre-installed) to a WP.com account
-3. By default, Gutenberg is the default editor so no action required there
+2. Connect Jetpack to the signed in WP.com account 
+3. Visit https://wordpress.com/settings/security, choose newly created site and enable "WordPress.com log in"
```

Comment:
I agree with this change because it keeps this test focused on Jetpack-connected self-hosted sites. Do you think a separate test should be added at some point to test the "enable Jetpack WP.com SSO flow"?

Reply:
I was thinking the same.  I'm not sure where the that test should live; in the Jetpack domain or Gutenberg. Regardless it's a good idea to have test coverage around that flow.

Reply:
For now I think it might be fine for it to live here, if you agree I'll add one here.
I made an issue to do that here: https://github.com/wordpress-mobile/test-cases/issues/81